Gds/src/fprime_gds/common/parsers/seq_file_parser.py

```diff
@@ -167,7 +166,7 @@ def parseAbsolute(timeStr):
                 dt = parseAbsolute(t)
                 # See if timezone was specified. If not, use UTC
                 if dt.tzinfo is not None:
-                    print("Using timezone %s" % dt.tzinfo.tzname())
+                    print("Using timezone %s" % dt.tzinfo.tzname())  # TODO: tzname method is unfilled
```

Comment:
This needs attention from maintainers to find out what is the raised issue here.

Reply:
OK